build.gradle

```diff
@@ -34,27 +25,24 @@ repositories {
         url "https://dl.bintray.com/ethereum/maven/"
     }
     maven {
-        url "https://research.rightmesh.io/artifactory/libs-local"
+        url "https://artifactory.rightmesh.io/artifactory/maven"
         credentials {
-            username artifactory_app_username
-            password artifactory_app_password
+            username rightmesh_artifactory_username
+            password rightmesh_artifactory_password
         }
     }
 }
 
 dependencies {
-    compile('io.left.rightmesh:lib-rightmesh-jre:BEN-SNAPSHOT')
+    compile 'io.left.rightmesh:lib-rightmesh-jre:BEN-SNAPSHOT'
```

Comment:
Do we have an official `0.10.0` build we can use?

Reply:
Not yet, but probably early next week?

Reply:
Looks like we do.